### DISCUSSION TITLE
Can't enable natural scrolling

### DISCUSSION THREAD

**Discussion by riley-martin** [WayfireWM/wayfire#1565] [Q&A] [answered]
I can't seem to enable natural scrolling with wayfire.
This is the input section of `~/.config/wayfire.ini`:
```
[input]
xkb_layout = us
xkb_variant = dvp
xkb_options = caps:swapescape
natural_scroll = true
```
I have tried setting and unsetting this option several times and logging out and back in
again, but it hasn't changed.
I am using a mouse, not a touchpad if that makes a difference.

**Comment by mepster** [comment, marked as answer]
I got it to work!

natural_scroll does not work. I read somewhere else it's not implemented (yet?)

Instead, in your .config/wayfire.ini after [input] you can add
mouse_scroll_speed=-1.0

You also have to hack /usr/share/wayfire/metadata/input.xml
Search for mouse_scroll_speed .
Remove the line <min>0.0</min>
This lets you set a negative mouse_scroll_speed.

Then log out / log in to reload the .xml file.

**Reply by 42-sol-137** [reply]
Hi mepster.

I also asked this in the Raspberry Pi forums.

I cannot find a file named .config/wayfire.ini and I'm allowing search of hidden files.  Might you be able to clue me in?

I can find and edit /usr/share/wayfire/metadata/input.xml without any problems.

Much appreciated.
